xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -195,9 +196,6 @@ ADDON_STATUS CPVRClient::Create(int iClientId)
   catch (std::exception &e) { LogException(e, __FUNCTION__); }
 
   m_bReadyToUse = bReadyToUse;
-
-  SetEPGTimeFrame(CSettings::GetInstance().GetInt(CSettings::SETTING_EPG_DAYSTODISPLAY));
```

Comment:
It is still used. https://github.com/xbmc/xbmc/blob/master/xbmc/pvr/PVRManager.cpp#L181